controllers/keystoneendpoint_controller.go

```diff
@@ -338,17 +341,19 @@ func (r *KeystoneEndpointReconciler) reconcileNormal(
 		instance.Spec.Endpoints,
 	)
 
-	util.LogForObject(helper, "Reconciled Endpoint normal successfully", instance)
+	logFromCtx.Info("Reconciled Endpoint normal successfully", "instance", instance)
 
 	return ctrl.Result{}, nil
 }
 
 func (r *KeystoneEndpointReconciler) reconcileEndpoints(
+	ctx context.Context,
 	instance *keystonev1.KeystoneEndpoint,
 	helper *helper.Helper,
 	os *openstack.OpenStack,
 ) error {
-	util.LogForObject(helper, "Reconciling Endpoints", instance)
+	logFromCtx := log.FromContext(ctx)
+	logFromCtx.Info("Reconciling Endpoints", instance)
```

Comment:
This causes a panic when you run the current patch. Info() needs to have one string argument and then pair of arguments, one string and one any. 

Specifically for this place `instance` is superfluous the logger automatically logs the same information already at the `KeystoneEndpoint` key:

```
2023-04-26T10:20:29+02:00	INFO	Reconciling Endpoints	{"controller": "keystoneendpoint", "controllerGroup": "keystone.openstack.org", "controllerKind": "KeystoneEndpoint", "KeystoneEndpoint": {"name":"glance-internal","namespace":"openstack"}, "namespace": "openstack", "name": "glance-internal", "reconcileID": "7390b221-6215-4990-8509-c7864d83c3d0", "instance": {"apiVersion": "keystone.openstack.org/v1beta1", "kind": "KeystoneEndpoint", "namespace": "openstack", "name": "glance-internal"}}
```